manifests/a/APC/PowerChutePersonalEdition/3.1.0/APC.PowerChutePersonalEdition.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: APC.PowerChutePersonalEdition
+PackageVersion: 3.1.0
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://download.schneider-electric.com/files?p_File_Name=PCPE_3.1.0.exe
+  InstallerSha256: 06CDB01B2E23ED72452FF090212BA72B5A250C16C41485ECD05D4AC9F75E25FF
+ManifestType: installer

```

Comment:
InstallerSwitches:
    Silent: /silent
    SilentWithProgress: /silent

Reply:
Switches are already included in the manifest as part of `InstallerType: nullsoft` which runs the manifest with `/S` from https://github.com/microsoft/winget-cli/blob/cf6f8e58195bf033a1a5872f7b800866cd27a586/src/AppInstallerCommonCore/Manifest/ManifestCommon.cpp#L362-L368

`/silent` is not a valid switch for nullsoft applications.